src/generators/web.cr

```diff
@@ -174,6 +177,14 @@ class LuckyCli::Generators::Web
           version: ~> 0.2
       DEPS_LIST
     end
+
+    if api_only? && generate_auth?
+      append_text to: "shard.yml", text: <<-DEPS_LIST
+
```

Comment:
If you rebase there should be a fix that adds a newline so I don't this this newline is necessary anymore